docs/intro/tutorial05.txt

```diff
@@ -457,7 +457,7 @@ class:
 
     def create_question(question_text, days):
         """
-        Creates a question with the given `question_text` published the given
```

Comment:
It reads correct to me as-is. Maybe "and published the given" (adding "and") would clarify the meaning for you?